app.js

```diff
@@ -1,11 +1,16 @@
 const persist = require('choo-persist')
 const mount = require('choo/mount')
 const log = require('choo-log')
+const css = require('sheetify')
```

Comment:
Just curious so I know for the future: What's the convention you follow for the order and naming of variables here?